sympy/stats/crv.py

```diff
@@ -31,6 +31,14 @@ class ContinuousDomain(RandomDomain):
     is_Continuous = True
 
     def as_boolean(self):
+        """
+        Return as is_Boolean
+
+        Raises
+        ========
+
+        NotImplementedError
```

Comment:
I don't think this should be mentioned. This is, as I understand it, a generic superclass method which should be overridden in subclasses. The only way someone would see this documentation is if the subclass doesn't implement the method, or if they are reading the documentation for the base class specifically. So the docstring should describe what the method should implement, and mention that it needs to be defined for subclasses.